definitions/npm/glamorous_v4.x.x/.flowconfig

```diff
@@ -0,0 +1,5 @@
+[libs]
```

Comment:
I'll rebase this out or remove it before this is done, this just makes development much more convenient than running tests.